docs/sources/installation/upgrading.md

```diff
@@ -85,7 +85,7 @@ sudo apt-get update
 
 #### Upgrade from binary .tar file
 
-If you downloaded the binary `.tar.gz` package, then you can just download and extract the new package and overwrite all your existing files. However, this might overwrite your config changes. 
+If you downloaded the binary `.tar.gz` package, then you can just download and extract the new package and overwrite all your existing files. However, this might overwrite your config changes.
```

Comment:
```suggestion
If you downloaded the binary `.tar.gz` package, you can just download and extract the new package and overwrite all your existing files. However, this might overwrite your config changes.
```

Reply:
These are unrelated so suggest adding them in separate PR

Reply:
OK @marefr 👍